src/appshell/qml/Preferences/internal/NoteInputSection.qml

```diff
@@ -70,6 +72,21 @@ BaseSection {
         }
     }
 
+    CheckBox {
+        id: colorChordsNotesBox
+        width: parent.width
+
+        text: qsTrc("appshell/preferences", "Color chord notes outside of usable pitch range")
```

Comment:
I'm not sure if this is good wording. This feature doesn't really have to do with "usable pitch range" ("range" implies that it would have something to do with the highest or lowest pitch that an instrument could possibly play but that is not what this is about).

How about something like just "Color unplayable chords"?
(I'd then also update the method names in the code everywhere to reflect how this feature is called in the UI, because otherwise it becomes confusing)